shared/js/background/startup.es6.js

```diff
@@ -90,6 +81,5 @@ function registerUnloadHandler () {
 }
 
 module.exports = {
-    onStartup,
```

Comment:
IIUC this is only needed as we're mixing cjs and esm right?

You're limiting startup to a single inclusion point via require; could we just convert this to ESM as only background.js needs it after this change.

Reply:
No, unfortunately not. This change is a prerequisite for https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/1464 since we now require `tab-manager` from startup with the recent changes that landed, we require `startup` from `declarative-net-request` already, and with my upcoming changes we need to require `declarative-net-request` from `tab-manager` too. I initially fixed it for just my case, but then I realised this is a trap that folks will continue falling in going forwards.